CONTRIBUTING.rst

```diff
@@ -0,0 +1,53 @@
+Contributing
+############
+
+Contributions are welcome and much appreciated!
+
+Development
+-----------
+
+To setup `ConferenceScheduler` for local development:
+
+1. Fork the repository on Github `<https://github.com/https://github.com/PyconUK/ConferenceScheduler/fork>`_
+2. Clone your fork to your local machine::
+
+    git clone git@github.com:your_name_here/ConferenceScheduler.git
+
+3. Create a branch for your work::
+
+    git checkout -b name-of-your-branch
+
+4. Commit your changes and push your branch to Github
+
+5. Create a Pull Request at `<https://github.com/PyconUK/ConferenceScheduler/pulls>`_
+
+
+Testing
+-------
+
+ConferenceScheduler includes a comprehensive test suite. Please ensure that you run all the tests before submitting a Pull Request.
+
+To setup your environment for testing:
+
+1. Install the necessary Python libraries::
+
+    pip install -r requirements.txt
```

Comment:
You wouldn't get the pep8 plugin (also listed in setup.py), so I reckon it's safer to use requirements.txt